translate/rus.json

```diff
@@ -71,7 +71,7 @@
 "bottom":"дно",
 "brackets":"древо",
 "brown":"коричневый",
-"busted openings":"забитые дебюты",
+"busted openings":"Неудачные дебюты",
```

Comment:
After a long think, I came to a conclusion that this translation suits best but it is still not perfect. Pop-up explanation window required.